packages/server/utils/analytics/analytics.ts

```diff
@@ -109,6 +109,10 @@ export type AnalyticsEvent =
   | 'Snackbar Viewed'
   // Join request
   | 'Join Request Reviewed'
+  // Suggest Groups
+  | 'Suggest Groups Generated'
```

Comment:
+1: Suggest**ed** Groups Generated?

Reply:
"Suggest Groups" is the noun; it's the name of the button. Generate is the verb, so that's in the past tense. I think it's the same as "Join Request Reviewed" - we wouldn't say "Join Requested Reviewed"

Reply:
But it's not the button that was generated, right? It's the groups that were generated and the groups are "suggested groups" (from ChatGPT). That's how I interpret at least. It's a +1 comment though so it's up to you!

Reply:
Hmm yeah, I think you're right! Updated 👍